templates/secrets/metadata-connection-secret.yaml

```diff
@@ -3,6 +3,8 @@
 #################################
 {{- if (and .Values.data.metadataConnection (not .Values.data.metadataSecretName)) }}
 {{- $host := .Values.data.metadataConnection.host | default (printf "%s-%s.%s.svc.cluster.local" .Release.Name "postgresql" .Release.Namespace) }}
+{{- $pgbouncer_host := (printf "%s-%s.%s.svc.cluster.local" .Release.Name "pgbouncer" .Release.Namespace) }}
+
```

Comment:
what do you think to simplify code using extract to variables?

Reply:
{{- $host := (ternary $pgbouncer_host $host .Values.pgbouncer.enabled) }}

Reply:
{{- $port := (((ternary .Values.ports.pgbouncer .Values.data.metadataConnection.port .Values.pgbouncer.enabled) | toString) }}

Reply:
{{- $db := (ternary (printf "%s-%s" .Release.Name "metadata") .Values.data.metadataConnection.db .Values.pgbouncer.enabled) }}

Reply:
wdyt @schnie ?

Reply:
Yeah, given the _monster_ that is the `connection` attribute, not having that needing all the `ternary` there would do wonders for the readability of that line.